lib/genesis/genesis.go

```diff
@@ -33,6 +34,7 @@ type Genesis struct {
 type Data struct {
 	Name       string
 	ID         string
+	ChainType  string
```

Comment:
@edwardmack  I added it here and above, is there somewhere else I need to add it?

Reply:
@noot no, I think this should be all set now.